addons/account/views/report_invoice.xml

```diff
@@ -156,6 +156,16 @@
                                             </t>
                                         </tr>
                                     </t>
+                                    <t t-if="o.amount_total == o.amount_untaxed">
```

Comment:
Not sure about this condition, maybe is better testing the size of : o.amount_by_group if it's zero.

```suggestion
                                    <t t-if="len(o.amount_by_group) == 0">
```